keypress/base.js

```diff
@@ -1,5 +1,17 @@
-console.log("Sanity Check: JS is working!");
+//could not get this one
 
```

Comment:
No need to paste in a solution from the solutions file if you didn't get it.  Besides this being important exercise for you, we are looking to get an idea of your level of understanding.  They aren't 'graded'.  If you don't know, just try your best. :)